testing/page-loading-spec.js

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
```

Comment:
Let's put this in a folder callled tests/ and call the file allPages.js. I'm not entirely sure what our naming convention going forward would be but I think we might have e2e tests describing different facets of the site, so the directory might contain allPages.js, learnerView.js, etc., with allPages.js just checking whether each page loads OK.